scripts/seeds/chef/job_seed.js

```diff
@@ -5,11 +5,11 @@ const fortnight = moment.duration(2, 'w');
 const month = moment.duration(1, 'M');
 
 exports.seed = function (knex) {
-  return knex('jobs').del() // Deletes ALL existing entries
+  return knex('jobs').where('accountId', 'CHEF-123').del() // Deletes specific account entries
 
     .then(function () {
       return knex('jobs').insert({
-        id: '100',
+        id: '900000100',
         accountId: 'CHEF-123',
         title: 'Sous Chef',
         employer: 'The Green Goblin',
```

Comment:
that is on purpose so when you see a diff it provides you with just the line that has changed instead of two lines where the comma is added.

Is not possible on JSON files as they would not be valid if done this way.